fedcloud_monitoring_tools/appdb.py

```diff
@@ -57,3 +59,11 @@ def get_vo_for_site(self, site):
         else:
             return []
         return [i["VO"] for i in data]
+
+    def all_vos(self):
```

Comment:
Why not using ops portal instead? as AppDB is set to leave, I would avoid adding much here

Reply:
this method is actually not used at the moment, but can we leave it there for future reference for the time being?

I just added a new class to get VOs from the Operations Portal.

Reply:
good, then better remove this code, no?